test/pipeline_test.dart

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:http/http.dart';
+import 'package:test/test.dart';
```

Comment:
`package:` imports for other packages should be separate from `package:` imports for the current package.